x-pack/plugins/observability/server/lib/rules/rules_service.ts

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
This is the same pattern the Infra team uses (having a separate Rules Service that gets instantiated from `server/plugin.ts`). APM and Synthetics by contrast do not have a separate `rules_service` but just register the rules directly in `server/plugin.ts`.

Both approaches are valid of course, so the following is subjective :)

By choosing this approach there will be 2 apps that use a separate rule_service and 2 that don't. If you don't introduce the concept of a rule_service here, then the division between implementations will be 1 app that has it, and 3 that don't.

Reply:
Good point. I was not in favour for this extra class but like you mentioned, I followed a lot what was done in the Infra package. Going to place this directly into the server plugin 👍🏻